Automattic/vip-go-mu-plugins
# Upgrading VaultPress to 2.2.1

## Description

This PR upgrades the VaultPress plugin to the latest minor revision (from 2.2.0 to 2.2.1). The upgrade has been made by dumping the contents of the plugin from WordPress.org: https://wordpress.org/plugins/vaultpress/

## Changelog Description

### Plugin Updated: VaultPress 2.2.1

We upgraded VaultPress 2.2.0 to 2.2.1.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Go to `wp-admin` > `Jetpack` > `VaultPress`
3. Check that the page loads correctly and the version number matches 2.2.1.
